src/app/store-stats/ClarityCharacterStat.m.scss

```diff
@@ -0,0 +1,68 @@
+@use '../variables' as *;
+@use '../dim-ui/tooltip-mixins' as *;
+
+.communityInsightSection {
+  @include tooltip-section-color($communityBlue);
```

Comment:
This mixin should be removing the tooltip section's top border, but isn't.

![dim-tooltip-section-top-border](https://github.com/DestinyItemManager/DIM/assets/17512262/23ba7b25-354f-4c51-8763-290f45ceaa5f)

Perhaps we tack on a `!important` [here](https://github.com/DestinyItemManager/DIM/blob/clarity-character-stats/src/app/dim-ui/_tooltip-mixins.scss#L22) to fix that?

```diff
-border-top-color: transparent;
+border-top-color: transparent !important;
```